packages/devtools-client-adapters/src/firefox/commands.js

```diff
@@ -71,38 +98,39 @@ function onNewBreakpoint(location, res) {
     column: actualLocation.column
   } : location;
 
-  return BreakpointResult({
+  return {
     id: bpClient.actor,
-    actualLocation: Location(actualLocation)
-  });
+    actualLocation
+  };
 }
 
-function removeBreakpoint(breakpointId) {
+function removeBreakpoint(breakpointId: BreakpointId) {
   const bpClient = bpClients[breakpointId];
-  bpClients[breakpointId] = null;
+  delete bpClients[breakpointId];
   return bpClient.remove();
 }
 
-function setBreakpointCondition(breakpointId, location, condition, noSliding) {
+function setBreakpointCondition(breakpointId: BreakpointId, location: Location,
+                                condition: boolean, noSliding: boolean) {
   let bpClient = bpClients[breakpointId];
-  bpClients[breakpointId] = null;
+  delete bpClients[breakpointId];
 
   return bpClient.setCondition(threadClient, condition, noSliding)
     .then(_bpClient => onNewBreakpoint(location, [{}, _bpClient]));
 }
 
-function evaluate(script, { frameId }) {
-  const params = frameId ? { frameActor: frameId } : {};
-  const deferred = defer();
-
-  tabTarget.activeConsole.evaluateJS(script, (result) => {
-    deferred.resolve(result);
-  }, params);
+type EvaluateParam = {
```

Comment:
I wasn't sure what this object really was so I just gave it this, but we should fill this in later.